tests/moduletool/test_install.py

```diff
@@ -780,6 +780,58 @@ def test_project_install_requirement_not_loaded(
     assert message in (rec.message for rec in caplog.records)
 
 
+@pytest.mark.parametrize_any("env_var", ["PIP_EXTRA_INDEX_URL", "PIP_INDEX_URL"])
+def test_pip_extra_index_env(
+    tmpdir: py.path.local,
+    modules_v2_dir: str,
+    snippetcompiler_clean,
+    monkeypatch,
+    env_var,
+) -> None:
+    """
+    Test that PIP_EXTRA_INDEX_URL/PIP_INDEX_URL is not set in the subprocess doing an install_from_index
+    and that it is not changed in the active env.
+
+    The installation fails with an ModuleNotFoundException
+    as the index https://pypi.org/simple is needed to install lorem~=0.0.1,
+    but it is only present in the active env in PIP_EXTRA_INDEX_URL/PIP_INDEX_URL which is not know by the
+    subprocess doing the pip install.
+    """
+    monkeypatch.setenv(env_var, "https://pypi.org/simple")
```

Comment:
This is not safe: if someone were to upload a malicious `inmanta-module-minimalv2module` package to pypi its `setup.py` would get executed on our Jenkins hosts and developer machines. So either we need to do this test with a module we have ownership over, or we must not use a public index, e.g. the `local_module_package_index` (name could be off, didn't check) fixture.